src/otx/core/data/module.py

```diff
@@ -24,6 +26,21 @@
     from .dataset.base import OTXDataset
 
 
+@dataclass
+class DataMetaInfo:
```

Comment:
It seems that we need to consider the `BaseClass` (maybe `BaseDataMeta`?) to broaden the tasks. What do you think ? In my case, I need to add additional `HLabelInfo` dataclass and it would be better if there is a baseclass to inherit.

Reply:
How about HLabelDataMetaInfo(DataMetaInfo)?

Reply:
I told @jaegukhyun to share `DataMetaInfo` across all tasks. `HLabelInfo` can be inherited from `DataMetaInfo`. However, `DataMetaInfo` seems not appealing name to me. We might have a better name in the future or this time by someone.